`quasar create project` error: node-waf: command not found (due to zlib dependency)

**Describe the bug**
In specific setup (npm @7 with verdaccio) `quasar create project` gives error: node-waf: command not found (zlib).

**To Reproduce**
Steps to reproduce the behavior:
1. Use verdaccio
2. Use npm@7
3. quasar create project
4. choose npm

**Expected behavior**
Create project without error

**Screenshots**
```
npm ERR! code 127
npm ERR! path /Users/richard/tmp/project/node_modules/zlib
npm ERR! command failed
npm ERR! command sh -c node-waf clean || true; node-waf configure build
npm ERR! sh: node-waf: command not found
npm ERR! sh: node-waf: command not found
```

**Platform (please complete the following information):**
Quasar Version: 2.0.1
@quasar/app Version: 3.0.1

Quasar mode:
  - [X] SPA

OS: Linux, macOS
Node: v16.4.1
NPM: 7.19.1
Yarn: 1.22.10

**Additional context**
The problem only occurs when:
* npm @7
* Verdaccio

When I use npm @6 or bypass Verdaccio (rename .npmrc) it works fine. It also works when I use Yarn but I rather stick with npm. For now I'm going back to npm @6.

Question: Does Quasar really need zlib at all? Everywhere I look it tells it is old and integrated into Node. Could this dependency be removed? That should solve the issue.

Further reference: https://github.com/verdaccio/verdaccio/discussions/2316

--- a/app/package.json
+++ b/app/package.json
@@ -112,8 +112,7 @@
     "webpack-chain": "6.5.1",
     "webpack-dev-server": "4.0.0-beta.3",
     "webpack-merge": "5.8.0",
-    "webpack-node-externals": "3.0.0",
-    "zlib": "1.0.5"
+    "webpack-node-externals": "3.0.0"
   },
   "engines": {
     "node": ">= 12.22.1",

